CMakeLists.txt

```diff
@@ -468,6 +468,8 @@ if(WIN32)
     add_custom_target(MsQuicEtw_HeaderBuild
         DEPENDS ${QUIC_BUILD_DIR}/inc/MsQuicEtw.h)
 
+    install(FILES ${QUIC_BUILD_DIR}/inc/MsQuicEtw.h DESTINATION include)
```

Comment:
Why is this necessary?

Reply:
Because it looking MsQuicEtw.h in the installation header folder. 
Now msquic::msquic_platform take the detention and targets from cmake config, it also needed this header.

Reply:
What is 'it' in this context? I don't understand why anything outside of MsQuic internally would need this file.

Reply:
quic_trance.h use this header. I will post the error in next comment.

Reply:
We shouldn't be using STDOUT by default. It's not performant.

Reply:
ok, I will remove this define manually from external cmake, and remove this includes from external include folder (meaning remove this new line)

Reply:
I remove this line.